tv_apps/smart-clock/manifest.webapp

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Smart Clock",
+  "type": "certified",
+  "role": "widget",
```

Comment:
There is no role called `widget`. We should change it to `system` or modify app-deck to skip this role.